sql/001-create-table-and-load-data.sql

```diff
@@ -2,12 +2,17 @@ DROP TABLE IF EXISTS fishes;
 
 CREATE TABLE fishes (
   id int unsigned AUTO_INCREMENT,
-  name VARCHAR(100) NOT NULL,
-  weight VARCHAR(100) NOT NULL,
-  price VARCHAR(100) NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  priceInYen VARCHAR(255) NOT NULL,
+  inventoryQuantity VARCHAR(255) NOT NULL,
```

Comment:
quantityは個数ですが、このカラムは重さを表すのでamountで良いかなと思います。
```suggestion
  inventoryAmount VARCHAR(255) NOT NULL,
```

Reply:
修正を行いました！
https://github.com/Moriwaki-9082/Java-10th-topic/pull/6/commits/591e89023423652bec08d766aee95d06ba6b797a#diff-4645cfd99037fcae3115a737492ea916503cd68df9d9c43ed42aa041be8c4c27L11-R13

Reply:
ok!